spec/features/ips/view_ip_addresses_spec.rb

```diff
@@ -37,6 +37,7 @@
       let(:address_two) { '123 Northern Street, Whitechapel' }
       let(:postcode_one) { 'SE4 4BK' }
       let(:postcode_two) { 'NE7 3UP' }
+      let!(:location) { create(:location, organisation: user.organisation) }
```

Comment:
You can do this without `let!`, which I think is better (feels like this spec needs some love in general, it's very long).

I pushed up a [possible solution on another branch](https://github.com/alphagov/govwifi-admin/blob/radius-secret-testing/spec/features/ips/view_ip_addresses_spec.rb).

Reply:
Makes more sense I like it